### ISSUE TITLE
No provider for InjectionToken Nebular Stepper Component

### ISSUE THREAD

**Issue by shreejibawa** [akveo/nebular#1669]
### Issue type

**I'm submitting a ...**  (check one with "x")

* [x] bug report
* [ ] feature request

### Issue description

**Steps to reproduce:**

I've Imported module `NbStepperModule` in my app.module.ts and trying to use stepper in one of the route components defined `app.routing` and it throws error.

**Related code:**
![image](https://user-images.githubusercontent.com/8849256/60440164-d86f1580-9c31-11e9-8843-7623537ba5f2.png)
![image](https://user-images.githubusercontent.com/8849256/60440180-e6249b00-9c31-11e9-8d81-bb3301814276.png)

And using stepper like:

```
<nb-stepper orientation="horizontal">
    <nb-step [label]="labelOne">
        <ng-template #labelOne>First step</ng-template>
        <h4>Step content #1</h4>
        <p class="lorem">
          Lorizzle ipsum dolizzle stuff fizzle, consectetuer adipiscing break it down.
          Nullizzle sapien velizzle, my shizz pimpin', shizzle my nizzle crocodizzle shut the shizzle up,
          gravida vizzle, dang.
        </p>
        <button nbButton disabled nbStepperNext>prev</button>
        <button nbButton nbStepperNext>next</button>
    </nb-step>
</nb-stepper>
```

Throws following error when navigated to that page where I've used stepper:

![image](https://user-images.githubusercontent.com/8849256/60440349-44517e00-9c32-11e9-873f-97daee75b8da.png)

### Other information:

**npm, node, OS, Browser**
```
Node: 8.9.3
npm: 5.2.0
Chrome / Windows 10
```

**Angular, Nebular**
```
Angular: ^8.0.2
Nebular: ^3.5.0
```

**Comment by nnixaa** [comment]
Hi @shreejibawa could you post a reproducible example to stackblitz?

**Comment by gemins** [comment]
Same error here!

npm, node, OS, Browser
```
Node: 12.4.0
npm: 6.9.0
Chrome / iOS
```

Angular, Nebular
```
Angular: 8.0.0-rc.5
Nebular: 3.6.
```

Maybe is some dependencies?

**Comment by gemins** [comment]
The link to open the component view have a same error!
From: https://akveo.github.io/nebular/3.6.1/docs/components/stepper/examples#nbsteppercomponent

Link to component showcase, get the error on console log.
https://akveo.github.io/nebular/3.6.1/example/stepper/stepper-showcase.component

:( i need complete this feature hurry up, i will upgrade all to version 4.

**Comment by shreejibawa** [comment]
@nnixaa please check following link for issue:

https://akveo.github.io/nebular/3.6.1/docs/components/stepper/examples#nbsteppercomponent

**Comment by yggg** [comment]
Fixed in 3.6.2

**Comment by shreejibawa** [comment]
@yggg thank you really appreciate it!
